### DISCUSSION TITLE
PCL 周边收集

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#3175] [讨论]
今天在 Bilibili 上看到了 [《当我使用Windows 10宣传片的方式打开PCL2》](https://www.bilibili.com/video/BV1Lb4y1N7z6)
挺有意思的，在这里开个帖
以后有看到啥和启动器有关的有趣事件可以发到此集合来

**Comment by ZS2147483647** [comment]
有时候会搞成PLC,我刚看完别人的视频`自己一打开就是,然后忘截图了,重启就没了
